installers-conda/build_installers.py

```diff
@@ -132,8 +132,8 @@
 specs = {
     "python": "=" + PY_VER,
     "spyder": "=" + SPYVER,
-    "paramiko": "",
-    "pyxdg": "",
+    "paramiko": "",  # ??? Do we need this?
+    "pyxdg": "",  # ??? Do we need this?
```

Comment:
* Paramiko is required to establish SSH connections on remote servers from Windows hosts.
* Pyxdg is necessary to detect the list of installed applications on Windows. That's used to associate file types to be open in certain applications in Files.

    ![image](https://github.com/spyder-ide/spyder/assets/365293/29f4860c-16ef-4af1-b6a7-23ea2a6658df)

Reply:
Thank you for looking at this. My question is related to whether they need to be explicitly listed here. I think that since these packages are already listed as Spyder dependencies elsewhere, I suspect that they do not need to be explicitly listed here for the runtime environment.